### PR TITLE
Enhancement tmcr deck plan sensor equipment

### PR DESCRIPTION
**Pull request by nick-knowles**

### CODE REVIEW

**Comment by skinkie** [comment]
This one is now mixing two things, which would not allow us to work on this issue until the other one has been resolved.

File: xsd/netex_framework/netex_reusableComponents/netex_sensorEquipment_version.xsd [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+				<Aggregation>main schema</Aggregation>
+				<Audience>e-service developers</Audience>
+				<Coverage>Europe</Coverage>
+				<Creator>First drafted for version 1.0 CEN TC278 WG3 SG6 Editor Nicholas Knowles. mailto:schemer@netex.org.uk</Creator>
+				<Date>
+					<Created>2023-02-06</Created>
+				</Date>
+				<Date>
+					<Modified>2023-02-06</Modified>
+          Definition of CycleStorage number of spaces corrected. Doc change only.
```
Comment (Aurige) [inline xsd/netex_framework/netex_reusableComponents/netex_sensorEquipment_version.xsd:19]: probably not relevant ;-)

File: xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+				<Aggregation>main schema</Aggregation>
+				<Audience>e-service developers</Audience>
+				<Coverage>Europe</Coverage>
+				<Creator>First drafted for version 1.0 CEN TC278 WG3 SG6 Editor Nicholas Knowles. mailto:schemer@netex.org.uk</Creator>
+				<Date>
+					<Created>2023-02-06</Created>
+				</Date>
+				<Date>
+					<Modified>2023-02-06</Modified>
+          Definition of CycleStorage number of spaces corrected. Doc change only.
```
Comment (Aurige) [inline xsd/netex_framework/netex_reusableComponents/netex_spotEquipment_version.xsd:19]: probably not relevant ;-)

**Review by Aurige — COMMENTED** [review]
Do we have an easy way to say: "_all spots have a sensor of this type_" ?
May also apply to entrance sensors

**Comment by ue71603** [comment]
@Aurige  with some more guidance, @trurlurl could also work here.

**Comment by ue71603** [comment]
done in #615

**Comment by trurlurl** [comment]
Removing "needs documentation update" label since closed without having been merged.

**Pull request closed without merging**